code/datums/shuttle.dm

```diff
@@ -503,7 +503,7 @@
 
 	var/area/space
 
-	space = get_space_area
+	space = get_space_area()
 	if(!space)
 		warning("There is no area at 1,1,2!")
```

Comment:
wait why is 1,1,2 used for shuttles? theres like a landmark way way north of it that says nothing is supposed to be above or to the left of it, what the fug